<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>View::NewMPSoCDialog</class>
 <widget class="QDialog" name="View::NewMPSoCDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>439</width>
    <height>160</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Add New MPSoC</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <property name="sizeConstraint">
    <enum>QLayout::SetFixedSize</enum>
   </property>
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="minimumSize">
      <size>
       <width>340</width>
       <height>0</height>
      </size>
     </property>
     <property name="title">
      <string>MPSoC Parameters</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <layout class="QFormLayout" name="formLayout">
        <item row="0" column="0">
         <widget class="QLabel" name="label">
          <property name="whatsThis">
           <string>The number of processor columns</string>
          </property>
          <property name="text">
           <string>Width (x)</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="QSpinBox" name="xSpinBox">
            <property name="minimumSize">
             <size>
              <width>48</width>
              <height>0</height>
             </size>
            </property>
            <property name="whatsThis">
             <string>The number of processor columns</string>
            </property>
            <property name="minimum">
             <number>2</number>
            </property>
            <property name="maximum">
             <number>16</number>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSlider" name="xHorizontalSlider">
            <property name="whatsThis">
             <string>The number of processor columns</string>
            </property>
            <property name="minimum">
             <number>2</number>
            </property>
            <property name="maximum">
             <number>16</number>
            </property>
            <property name="pageStep">
             <number>2</number>
            </property>
            <property name="value">
             <number>2</number>
            </property>
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="label_2">
          <property name="whatsThis">
           <string>The number of processor rows</string>
          </property>
          <property name="text">
           <string>Height (y)</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <item>
           <widget class="QSpinBox" name="ySpinBox">
            <property name="minimumSize">
             <size>
              <width>48</width>
              <height>0</height>
             </size>
            </property>
            <property name="whatsThis">
             <string>The number of processor rows</string>
            </property>
            <property name="minimum">
             <number>2</number>
            </property>
            <property name="maximum">
             <number>16</number>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSlider" name="yHorizontalSlider">
            <property name="whatsThis">
             <string>The number of processor rows</string>
            </property>
            <property name="minimum">
             <number>2</number>
            </property>
            <property name="maximum">
             <number>16</number>
            </property>
            <property name="pageStep">
             <number>2</number>
            </property>
            <property name="value">
             <number>2</number>
            </property>
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="label_3">
          <property name="whatsThis">
           <string>The Heuristic for task allocation determination</string>
          </property>
          <property name="text">
           <string>Heuristic</string>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QComboBox" name="comboBox">
          <property name="whatsThis">
           <string>The Heuristic for task allocation determination</string>
          </property>
         </widget>
        </item>
        <item row="3" column="0">
         <widget class="QLabel" name="label_4">
          <property name="text">
           <string>Master:</string>
          </property>
         </widget>
        </item>
        <item row="3" column="1">
         <widget class="QLabel" name="masterAddr">
          <property name="text">
           <string>0, 0</string>
          </property>
         </widget>
        </item>
        <item row="4" column="0" colspan="2">
         <layout class="QGridLayout" name="gridLayout"/>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>View::NewMPSoCDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>286</x>
     <y>123</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>132</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>View::NewMPSoCDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>312</x>
     <y>123</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>132</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>xSpinBox</sender>
   <signal>valueChanged(int)</signal>
   <receiver>xHorizontalSlider</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>229</x>
     <y>43</y>
    </hint>
    <hint type="destinationlabel">
     <x>178</x>
     <y>46</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>xHorizontalSlider</sender>
   <signal>valueChanged(int)</signal>
   <receiver>xSpinBox</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>213</x>
     <y>36</y>
    </hint>
    <hint type="destinationlabel">
     <x>234</x>
     <y>43</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>yHorizontalSlider</sender>
   <signal>valueChanged(int)</signal>
   <receiver>ySpinBox</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>179</x>
     <y>79</y>
    </hint>
    <hint type="destinationlabel">
     <x>239</x>
     <y>77</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>ySpinBox</sender>
   <signal>valueChanged(int)</signal>
   <receiver>yHorizontalSlider</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>236</x>
     <y>70</y>
    </hint>
    <hint type="destinationlabel">
     <x>189</x>
     <y>74</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
